dronegen/windows.go

```diff
@@ -0,0 +1,226 @@
+// Copyright 2021 Gravitational, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import "path"
+
+const (
+	perBuildWorkspace   = `$Env:WORKSPACE_DIR/$Env:DRONE_BUILD_NUMBER`
+	windowsToolchainDir = perBuildWorkspace + `/toolchains`
+	perBuildTeleportSrc = perBuildWorkspace + "/go/src/github.com/gravitational/teleport"
+	perBuildWebappsSrc  = perBuildWorkspace + "/go/src/github.com/gravitational/webapps"
+
+	// Hardcoded tool versions that we would normally pull from the makefile,
+	// but unfortunately our Makefiles use too many POSIX-isms for us to use on
+	// Windows, so for now we will just say the versions we want here.
+
+	windowsNodeVersion = "16.13.2"
+)
+
+func newWindowsPipeline(name string) pipeline {
+	p := newExecPipeline(name)
+	p.Workspace.Path = path.Join("C:/Drone/Workspace", name)
```

Comment:
Same - does this actually work? I would have expected the following:
```suggestion
	p.Workspace.Path = path.Join(`C:\Drone\Workspace`, name)
```

Reply:
Yes - see https://drone.platform.teleport.sh/gravitational/teleport/14412